gradio/processing_utils.py

```diff
@@ -176,9 +180,11 @@ def encode_pil_to_bytes(pil_image, format="png"):
             pil_image.save(output_bytes, format, **params)
         return output_bytes.getvalue()
 
+hash_seed = str(uuid.uuid4()).encode("utf-8")
```

Comment:
Right now, `hash_seed` is refreshed every time someone restarts their Python session and reimports Gradio. I think this could lead to a lot of unnecessary files created locally. Imagine someone has Automatic1111 running locally and they launch the app with a bunch of images. New temporary copies will get saved each time the app is launched.

Instead, what about creating a `hash_seed` when Gradio is first installed and saving it in `launches.json` (or a different file name but the point is that it doesn't change for the user while Gradio remains installed)